ruby/lib/ci/queue/redis/worker.rb

```diff
@@ -11,6 +11,157 @@ class << self
       self.requeue_offset = 42
 
       class Worker < Base
+        class ErrorReport
```

Comment:
TODO: Find new location for `ErrorReport` and `FailureFormatter`. This may break public API?